examples/dedicated_vmseries_and_autoscale/example.tfvars

```diff
@@ -0,0 +1,289 @@
+          }
+        }
+      }
+      "public" = {
+        name = "public-rt"
+        routes = {
+          "mgmt_blackhole" = {
+            address_prefix = "10.0.0.0/28"
+            next_hop_type  = "None"
+          }
+          "private_blackhole" = {
+            address_prefix = "10.0.0.16/28"
+            next_hop_type  = "None"
+          }
+        }
+      }
+    }
+    subnets = {
+      "management" = {
+        name                            = "mgmt-snet"
+        address_prefixes                = ["10.0.0.0/28"]
+        network_security_group          = "management"
+        route_table                     = "management"
+        enable_storage_service_endpoint = true
+      }
+      "private" = {
+        name             = "private-snet"
+        address_prefixes = ["10.0.0.16/28"]
+        route_table      = "private"
+      }
+      "public" = {
+        name                   = "public-snet"
+        address_prefixes       = ["10.0.0.32/28"]
+        network_security_group = "public"
+        route_table            = "public"
+      }
+      "appgw" = {
+        name             = "appgw-snet"
+        address_prefixes = ["10.0.0.48/28"]
+      }
+    }
+  }
+}
+
+natgws = {
+  "natgw" = {
+    name              = "public-natgw"
+    vnet_key          = "transit"
+    subnet_keys       = ["public", "management"]
+    create_pip        = false
+    create_pip_prefix = true
+    pip_prefix_length = 29
+  }
+}
+
+
+
+# --- LOAD BALANCING PART --- #
+load_balancers = {
+  "public" = {
+    name                        = "public-lb"
+    network_security_group_name = "example-public-nsg"
+    network_security_allow_source_ips = [
+      #  "x.x.x.x", # Put your own public IP address here  <-- TODO to be adjusted by the customer
+      "0.0.0.0/0",
+    ]
+    frontend_ips = {
+      "palo-lb-app1-pip" = {
+        create_public_ip = true
+        in_rules = {
+          "balanceHttp" = {
+            protocol = "Tcp"
+            port     = 80
+          }
+        }
+      }
+    }
+  }
+  "private" = {
+    name = "private-lb"
+    frontend_ips = {
+      "ha-ports" = {
+        vnet_key           = "transit"
+        subnet_key         = "private"
+        private_ip_address = "10.0.0.30"
+        in_rules = {
+          HA_PORTS = {
+            port     = 0
+            protocol = "All"
+          }
+        }
+      }
+    }
+  }
+}
+
+appgws = {
+  "public" = {
+    name       = "public-appgw"
+    vnet_key   = "transit"
+    subnet_key = "appgw"
+    capacity   = 2
+    rules = {
+      "minimum" = {
+        priority = 1
+        listener = {
+          port = 80
+        }
+        rewrite_sets = {
+          "xff-strip-port" = {
+            sequence = 100
+            request_headers = {
+              "X-Forwarded-For" = "{var_add_x_forwarded_for_proxy}"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+
+
+# --- VMSERIES PART --- #
+application_insights = {}
+
+vmseries_version = "10.2.3"
```

Comment:
If we want to deploy 2 pairs of VM-Series with 2 different versions, then it's not possible. What do you think about defining PANOS version for each VM-Series group (inbound, outbound etc.) ?